app/models/activity_insight_oa_file.rb

```diff
@@ -12,14 +12,26 @@ class ActivityInsightOAFile < ApplicationRecord
 
   mount_uploader :file_download_location, ActivityInsightFileUploader
 
-  scope :ready_for_download, -> {
+  scope :oa_type_w_no_locations, -> {
```

Comment:
oooh, I think that this scope is going to need to be changed per #797. I guess you could either do that here or make it a separate issue and wait until after this is merged.

cc @Smullz622

Reply:
Ah right.  I'll do that here.